src/anchor.js

```diff
@@ -28,7 +29,43 @@ const aggregators = {
             r += part.r
         }
         return new Point(x / len, y / len, r / len)
-    }
+    },
+    intersect: (config, name, parts) => {
+        const get_line_from_point = (point, axis, offset=1000) => {
```

Comment:
My main issue with this PR is that it requires users to specify extra axes for how points should be "converted" to lines, while completely disregarding the rotation aspect of anchors, which should already be unambiguous. We should just take the point's x/y as `p1`, then project it towards the rotation it's "facing" to get `p2`. So no need for `axis1` and `axis2` params.

Reply:
> We should just take the point's x/y as `p1`, then project it towards the rotation it's "facing" to get `p2`. So no need for `axis1` and `axis2` params.

But what do you consider as the facing axis? The X one?

Reply:
It should also be noted that `affect` already supports scenario #1 becauce one could anchor to both A and B point refs, and affect X and Y respectively.

Reply:
I might be misunderstanding something, but in my view, there **are** no two axes. Where do these X/Y choices even come from? Points here are a triplet of `[x, y, r°]`, and the rotation degrees unambiguously declare where a point is facing. So each point also implies a directed line. Two points, two lines. Two non-parallel lines have exactly one intersection.

Reply:
I am also confused by this feedback and do not understand how I should figure out along which axis I should create the lines. Especially if there can be intersections on multiple points.

I am fine with default values for the axis parameters and would propose x and y. That seems to be the most common use case in my mind and will get an intersection even if the points have no rotation.

But I think not having default values is better, because it offers a way to self-document. If someone is just writing the config without reading the documentation, the asserts will tell them the parameters that are requiered and what the acceptable value are.

Reply:
Ooh, I see the confusion now! In this case, yes, I mean automatic Y axis for both inputs. If r = 0°, then the point is "looking up". It may help to take a look at these examples in the new (in-progress) docs: https://deploy-preview-12--ergogen.netlify.app/points#examples

As for "opposite directions", we can get around it by extending into both directions, so this last A vs. B intersection would fall to the left of them both.

Reply:
Excellent visualizations, indeed. Now we're definitely on the same page about what each of us mean. Only I disagree with what you mean :sweat_smile: Wanting this bottom right point as the intersection **should** require the user to rotate p2 by -90°, which they can easily do as the `parts` of the aggregate can be full anchor declarations themselves - so it's just an extra `rotate`. Also, them **not** rotating this -90° **should** lead to the intersection on the upper left by default, even tho the points are not facing that way. This whole "facing" thing is just to establish the point's line, and wherever the two lines meet, that's the intersection. In my view, it's exactly these axis parameters that make things ambiguous. Points already define a line, and intersecting two of these lines is well defined.